pylutron/__init__.py

```diff
@@ -1104,19 +1127,29 @@ class Event(LutronEvent):
     """
     OCCUPANCY = 1
 
-  def __init__(self, lutron, area, uuid):
-    super(OccupancyGroup, self).__init__(lutron, 'Occ {}'.format(area.name), uuid)
+  def __init__(self, lutron, group_number, uuid):
+    super(OccupancyGroup, self).__init__(lutron, None, uuid)
```

Comment:
by the way, why did you drop the name completely? Don't we need something for HASS?

Reply:
We override name() from LutronEntity, and we don't have the area's name until we parse the areas and bind them back to the occupancy groups later.